lints.toml

```diff
@@ -24,6 +24,7 @@ deny = [
 ]
 allow = [
     'unused_results',
+	'clippy::string_add',
```

Comment:
this one is allowed by default, and unlikely to change I think

Reply:
It wasn't enabled in our codebase, because of `clippy::restriction`.